docker-compose.dcproj

```diff
@@ -7,6 +7,8 @@
     <DockerLaunchBrowser>True</DockerLaunchBrowser>
     <DockerServiceUrl>http://localhost:5102/swagger</DockerServiceUrl>
     <DockerServiceName>enrolling.api</DockerServiceName>
+    <DockerServiceUrl>http://localhost:5103/swagger</DockerServiceUrl>
```

Comment:
The file only take one `DockerServiceName` and `DockerServiceUrl`. Please remove newly added service config

Reply:
Do you mean to remove the following section?
```
   <DockerServiceName>enrolling.api</DockerServiceName>
    <DockerServiceUrl>http://localhost:5103/swagger</DockerServiceUrl>
```

Reply:
You can reset any file by running the command

```bash
git checkout master docker-compose.dcproj
```

This will reset any changes you made to this file in your branch.

Reply:
You do not need to modify this file

Reply:
Modified